doc/code/qml_tape.rst

```diff
@@ -0,0 +1,160 @@
+qml.tape
+========
+
+.. warning::
+
+    The new PennyLane tape mode is **experimental**, and does not currently have feature-parity with
+    the existing QNode. `Feedback and bug reports
+    <https://github.com/PennyLaneAI/pennylane/issues>`__ are encouraged and will help improve the
+    new tape mode.
+
+    Tape mode can be enabled globally via the :func:`~.enable_tape` function, without changing your
+    PennyLane code:
+
+    >>> qml.enable_tape()
+
+    Once enabled, tape mode can be disabled via :func:`~.disable_tape`.
+
+
+Tape-mode QNodes
+----------------
+
+The PennyLane tape module provides a new QNode class, rewritten from the ground-up,
+that uses a :class:`~.QuantumTape` to represent the internal variational quantum circuit.
+Tape mode provides several advantanges over the standard PennyLane QNode.
+
+* **No more Variable wrapping**: In tape mode, QNode arguments no longer become :class:`~.Variable`
+  objects within the QNode; arguments can be easily inspected, printed, and manipulated.
+
+  >>> dev = qml.device("default.qubit", wires=1)
+  >>> @qml.qnode(dev)

```

Comment:
Would these need to be included till tapes are experimental?
```suggestion
  >>> @qml.tape.qnode(dev)

```

Reply:
Oh, this was intentional --- I wrote this section with the assumption that the user had execute `qml.enable_tape()`. I am trying to position that as the main supported way of using tape mode.

Reply:
Oh got ya! Might be just me, but could be good to include it so that the examples are stand-alone? KInd of liked that later examples had it and could recreate the output just by running the snippet

Reply:
Good point, I'll add `qml.enable_tape` to the top of each example

Reply:
done